_layouts/default.html

```diff
@@ -75,8 +75,8 @@
             <hr>
             <h2 class="text-delta">Table of contents</h2>
             <ul>
-              {% for child in pages_list %}
-                {% if child.parent == page.title and child.title != page.title %}
+              {% for child in children_list %}
```

Comment:
The variable `children_list` is assigned the children lists of different pages in `nav.html`, and needs to be updated here to the list of children of the current page:

```suggestion
              {%- assign children_list = pages_list | where: "parent", node.title -%}
              {% for child in children_list %}
```

Without the above change, the Table of Contents, automatically appended to pages with children, is always empty.